ui/shared/TransactionStackGroup.qml

```diff
@@ -0,0 +1,61 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import "../imports"
+
+Rectangle {
+    id: root
+    property string headerText
+    property string footerText
+    property var isValid: true
+    property var validate: function() {
+        let isValid = true
+        for (let i=0; i<children.length; i++) {
+            const component = children[i]
+            if (component.hasOwnProperty("validate") && typeof component.validate === "function") {
+                isValid = component.validate()
+            }
+        }
+        root.isValid = isValid
+        return isValid
+    }
+    color: Style.current.background
+    function reset() {
+        for (let i=0; i<children.length; i++) {
+            const component = children[i]
+            try {
+                if (component.hasOwnProperty("resetInternal") && typeof component.resetInternal === "function") {
+                    component.resetInternal()
+                }
+                if (component.hasOwnProperty("reset") && typeof component.reset === "function") {
+                    component.reset()
+                }
+            } catch (e) {
+                console.warn("Error resetting component", i, ":", e.message)
+                continue
+            }
+        }
+    }
```

Comment:
As mentioned in the other comment, I think it'd be cool to have a `FormGroup` that handles validation + reset

Reply:
Done.